Module3_8/Module3_8/Program.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System;
+
+namespace Module3_8
+{
+    class Program
+    {
+        static void Main()
+        {
+            double startOfLine = -2;
+            double endOfLine = 7;
+            double epsilon = 0.001;
+            double x = (endOfLine + startOfLine) / 2;
+
+            while (Math.Abs(F(x)) > epsilon)
+            {
+                if (F(x) > 0)
+                    endOfLine = x;
+                else
+                    startOfLine = x;
+                x = (startOfLine + endOfLine) / 2;
+            }
+
+            Console.WriteLine($"X = {x}");
+            Console.WriteLine($"F(x) = {F(x)}\n");
+            Console.WriteLine("Press any key\n");
+            Console.ReadKey();
+
+            // Second part of the task. Array in spiral.
+
+            int size = 4;
+            int[,] testArray = new int[size, size];
+
+            FillArrayInSpiral(testArray);
+            ShowArrayNumbers(testArray);
+
+            Console.ReadKey();
+        }
+
+
+        static double F(double x)
```

Comment:
what is the method ? I didnt understand at all what the method do. You should write self documented code